src/julia-syntax.scm

```diff
@@ -2599,6 +2600,10 @@
         ((eq? (car e) 'local) '(null)) ;; remove local decls
         ((eq? (car e) 'local-def) '(null)) ;; remove local decls
         ((eq? (car e) 'implicit-global) '(null)) ;; remove implicit-global decls
+	((eq? (car e) 'require-existing-local)
+	 (if (not (memq (cadr e) env))
+	     (error "no outer variable declaration exists for \"for outer\""))
+	 '(null))
```

Comment:
alignment here is off: need to use spaces, not tabs

Reply:
Oops --- on a different machine that doesn't have my usual emacs settings.